sdks/cpp/common/include/ParamAccessor.h

```diff
@@ -42,6 +42,8 @@
 #include <typeinfo>
 #include <typeindex>
 
+#define AUTHZ_DISABLED "__AUTHZ_DISABLED__"
```

Comment:
hey! 1970's code!
in C++ we prefer to write const std::string kAuthzDisabled("__AUTHZ_DISABLED__"); 
why?
that's a good research topic for you. something to do with being able to pass by reference, take addresses etc.

Reply:
Fixed